examples/encoding/plot_continuous_encoder.py

```diff
@@ -0,0 +1,111 @@
+    fields of auditory and visual neurons from their responses to
+    natural stimuli. Network 12, 289-316 (2001).
+
+    Willmore, B. & Smyth, D. Methods for first-order kernel
+    estimation: simple-cell receptive fields from responses to
+    natural scenes. Network 14, 553-77 (2003).
+"""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#
+# License: BSD (3-clause)
+
+import matplotlib.pyplot as plt
+
+import mne
+import numpy as np
+from mne.datasets import sample
+from mne.encoding import SampleMasker, get_coefs, get_final_est, FeatureDelayer
+from sklearn.pipeline import make_pipeline
+from sklearn.linear_model import Ridge
+from sklearn.preprocessing import StandardScaler
+
+###############################################################################
+# Load and preprocess data
+data_path = sample.data_path()
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+raw = mne.io.read_raw_fif(raw_fname, preload=True).pick_types(
+    meg=False, stim=False, eeg=True).filter(1, None, method='iir')
+seed_channel = 'EEG 011'  # Choosing a frontal electrode
+ix_seed = mne.pick_channels(raw.ch_names, [seed_channel])[0]
+ix_targets = np.setdiff1d(range(len(raw.ch_names)), [ix_seed])
+
+# Set up encoding model variables
+X = raw[:][0][ix_targets]
+y = raw[:][0][ix_seed]
+n_targets, n_times = X.shape
+
+###############################################################################
+# Fit the model / Make predictions
+# Define model preprocessing and parameters
+ix_train = np.arange(n_times // 2)
+ix_test = np.arange(n_times // 2, n_times)
+
+# This creates time-lagged versions of the data (in seconds)
+delay_step = .01  # Time between delays (in seconds)
+delayer = FeatureDelayer(sfreq=raw.info['sfreq'],
+                         delays=np.arange(-.1, .1, delay_step))
+info_new = raw.copy().drop_channels([seed_channel]).info.copy()
+info_new['sfreq'] = 1. / delay_step
+
+# This is the final estimator that will be called after masking data
+estimator = make_pipeline(StandardScaler(), Ridge(alpha=1.))
+# This allows us to use a subset of indices for training / prediction
+masker = SampleMasker(estimator, ixs=ix_train, ixs_pred=ix_test)
+# Finally, place a delayer first so that samples are delayed before being split
+pipeline = make_pipeline(delayer, masker)
+
+# Fit / predict with the model
+pipeline.fit(X.T, y)  # estimators expect data of shape (n_samples, n_features)
+y_pred = pipeline.predict(X.T)
+
+###############################################################################
+# Plot raw data
+fig, (ax_raw, ax_pred) = plt.subplots(2, 1, figsize=(8, 8))
+plt_times = raw.times[:200]
+for idata, alpha, lw, color in zip([X, y], [.2, .8], [1, 3], ['k', 'r']):
+    ax_raw.plot(plt_times, idata[..., :200].T, color=color, alpha=alpha, lw=lw)
+ax_raw.set_title('Continuous training data')
+ax_raw.set_xlim([0, plt_times[-1]])
+ax_raw.legend([ax_raw.lines[0], ax_raw.lines[-1]], ['Inputs', 'Seed'])
```

Comment:
I was having a hard time getting this to work properly, since the first call plots lots of lines and the second call only plots one line, which is why I was specifying them manually. I agree your way is more clear but not sure how to debug it...